app/javascript/packages/components/troubleshooting-options.tsx

```diff
@@ -19,13 +19,16 @@ interface TroubleshootingOptionsProps {
   options: TroubleshootingOption[];
 
   isNewFeatures?: boolean;
+
+  divider?: boolean;
 }
 
 function TroubleshootingOptions({
   headingTag = 'h2',
   heading,
   options,
   isNewFeatures,
+  divider = true,
```

Comment:
Can we add a test case(s) that the expected class is added / not added depending on this prop?